src/tox_lsr/test_scripts/runansible-doc.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# A shell wrapper around ansible-doc.
+
+set -u
+
+ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "$SCRIPTDIR/utils.sh"
+
+if [ "${LSR_ROLE2COLL_NAMESPACE:-}" = "" -o "${LSR_ROLE2COLL_NAME:-}" = "" ]; then
+    lsr_info "${ME}: Collection format only."
+    exit 1
+fi
+
+LSR_ROLE2COLL_MODULES="$MY_LSR_TOX_ENV_DIR"/ansible_collections/"$LSR_ROLE2COLL_NAMESPACE"/"$LSR_ROLE2COLL_NAME"/plugins/modules
+
+rval=0
+if [ -d "$LSR_ROLE2COLL_MODULES" ]; then
+  modules=()
+  for module_path in $( ls "$LSR_ROLE2COLL_MODULES" ); do
+    module_file=$( basename $module_path )
+    module="${module_file%%.*}"
+    modules+=("$LSR_ROLE2COLL_NAMESPACE"."$LSR_ROLE2COLL_NAME"."$module")
+  done
+
+  for module in "${modules[@]}"; do
+    lsr_info "${ME}: Checking $module ..."
+    ANSIBLE_COLLECTIONS_PATHS="$MY_LSR_TOX_ENV_DIR" ansible-doc -M "$LSR_ROLE2COLL_MODULES" \
+      -vvv --type module "$module"; r=$? || :
```

Comment:
The `|| :` here is in the wrong place - it needs to be immediately after the command that may return a non-zero exit code - so
```bash
ANSIBLE_COLLECTIONS_PATHS="$MY_LSR_TOX_ENV_DIR" ansible-doc -M "$LSR_ROLE2COLL_MODULES" \
      -vvv --type module "$module" || r=$?
```

Reply:
Thanks, @richm!

It looks your previous advice https://github.com/linux-system-roles/tox-lsr/pull/15#discussion_r569062854 works well with "-euo pipefail".  So, I don't need to do this error ignoring. :) I'm updating this pr soon.